pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -127,16 +127,54 @@ message NamespaceOption {
     optional bool host_ipc = 3;
 }
 
+// PodSecurityContext holds pod-level security configuration that will be applied to a container.
```

Comment:
@feiskyer is right.  You can check out the proposal for more info: https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/pod-security-context.md

Of course, I realize now that I never moved that proposal to 'designs', which is what you're supposed to do when you implement a proposal :) So, it'll be in designs soon.